src/fw/drivers/wscript_build

```diff
@@ -465,14 +465,15 @@ if bld.is_silk() and not bld.env.QEMU:
     )
 
 if bld.is_asterix() or bld.is_obelix():
-    # FIXME(ASTERIX,OBELIX): provide working implementation
     bld.objects(
         name='driver_lsm6dso',
         source=[
-            'stubs/accel.c',
+            'imu/lsm6dso/lsm6dso.c',
         ],
         use=[
             'fw_includes',
+            'lsm6dso',
+            'lsm6dso_includes',
```

Comment:
name library `hal_lsm6dso`, no need to use `_includes` if original target exports includes.